packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -22,22 +30,52 @@ export function collectDependencies({ onlyNative, outputDir, widgetName }) {
             const resolvedPackagePath = await resolvePackage(
                 source,
                 dirname(importer ? importer : rollupOptions.input[0])
-            );
-            if (resolvedPackagePath && (!onlyNative || (await hasNativeCode(resolvedPackagePath)))) {
-                if (!managedDependencies.includes(resolvedPackagePath)) {
-                    managedDependencies.push(resolvedPackagePath);
+            ).catch(e => {
+                console.warn(e.message);
+                return null;
```

Comment:
Howcome you are catching and silencing this error?

Reply:
It's a bit scary as this is a change in behaviour

Reply:
I removed this part.